juju/juju
# Add caas provisioner facades and wire up the worker

## Description of change

Add the necessary caas provisioner facades and wire up the caas provisioner.
A "no-op" caas model upgrade worker is also needed to allow the upgrade gate to be closed to allow the worker to start up.

## QA steps

deploy kubernetes
add cass cloud
add caas model using the caas cloud
deploy any charm
use kubectl to see that a pod named after the charm is started
